.ado.yml

```diff
@@ -45,25 +45,17 @@ jobs:
         solution: PerfView.sln
         configuration: Debug
 
-    - task: VisualStudioTestPlatformInstaller@1
-      displayName: Install Latest Stable VSTest
-      inputs:
-        packageFeedSelector: 'nugetOrg'
-        versionSelector: 'specificVersion'
-        testPlatformVersion: '17.6.3'
-
     - task: VSTest@2
       displayName: 'Run Tests'
       inputs:
         testAssemblyVer2: |
-         **\LinuxTracing.Tests.dll
-         **\Tests.dll
-         **\TraceEventTests.dll
-         **\PerfViewTests.dll
-         !**\*TestAdapter.dll
-         !**\obj\**
+         **\bin\**\LinuxTracing.Tests.dll
+         **\bin\**\Tests.dll
+         **\bin\**\TraceEventTests.dll
+         **\bin\**\PerfViewTests.dll
```

Comment:
Can also make this just `**\bin\**\*Tests.dll`, but wanted to give the UI tests a chance to run last, because they have the highest chance of killing the whole run.